charts/incubator/dumbassets/values.yaml

```diff
@@ -0,0 +1,39 @@
+image:
+  repository: dumbwareio/dumbassets
+  pullPolicy: IfNotPresent
+  tag: 1.0.11@sha256:b12310240508ee26acd4adbbf4b3e42303a4c5918b094daf0d225b3ae00cad93
+securityContext:
+  container:
+    runAsNonRoot: false
+    readOnlyRootFilesystem: false
+    runAsUser: 0
+    runAsGroup: 0
+service:
+  main:
+    ports:
+      main:
+        port: 3000
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          probes:
+            liveness:
+              type: http
+              path: /
+            readiness:
+              type: http
+              path: /
+            startup:
+              type: http
+              path: /
+          env: {}
```

Comment:
can you add all the env variables here?

Reply:
like normal env on line 44, no need for a custom object

Reply:
```Fail: Container - Environment Variable [TZ] in [env] tried to override the Environment Variable that is already defined in [fixedEnv]```


@xstar97 not sure why. Some other charts are also using the TZ in their env variables...